### PR TITLE
Multibase strings

### LINKED ISSUE #5: Parse digest as Multibase instead of plain bytes
digests may be encoded using multibase https://github.com/multiformats/multibase

as parse for multibase would be necessary for parsing such digests.

### PR DESCRIPTION
**Pull request by andrevidela**
fix #5

### CODE REVIEW

File: Data/Multibase.idr [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  SBase2     : BaseSymbol 1
+  SBase8     : BaseSymbol 7
+  SBase10    : BaseSymbol 9
+  SBase16    : BaseSymbol 15
+  SBase32    : BaseSymbol 31
+  SBase58btc : BaseSymbol 57
+  SBase64    : BaseSymbol 63
+  
+public export
+record MultibaseDigest (length : Nat) (n : Nat) where
```
Comment (epost) [inline Data/Multibase.idr]: This seems to be the core type of the whole thing. Would it be helpful to add a doc comment with an example? And perhaps a link to something like https://github.com/multiformats/multihash#format?

**Review by epost — APPROVED** [review]
Looks great! I'd like to suggest to add a link to the tests to the README, because they're very instructive.

File: Data/Multibase.idr (hunk 1 above)
Comment (andrevidela) [reply Data/Multibase.idr]: Good idea! Will do

**Comment by andrevidela** [comment]
Thanks for taking a look @epost I think I will write proper tests before merging this pull request

**Comment by andrevidela** [comment]
Alright this can parse encoded strings but cannot decode them I'll take care of this in the next pull request.

**Pull request merged**